plugins/mumble_plugin_utils.h

```diff
@@ -17,7 +17,8 @@
 // allows the string to be safely used
 // when constructing JSON documents via
 // string concatenation.
```

Comment:
Add a third paragraph:

    Finally, escape ensures that the given string is NUL-terminated
    by always setting the last byte of the input string to the value 0.